packages/modelviewer.dev/examples/loading/index.html

```diff
@@ -0,0 +1,275 @@
+          <div class="heading">
+            <h2 class="demo-title">Poster with invalid model</h2>
+            <h4></h4>
+          </div>
+          <example-snippet stamp-to="demo-container-4" highlight-as="html">
+            <template>
+    <model-viewer poster="../../assets/poster-astronaut.png" src="i-do-not-exist.glb" alt="An invalid model"></model-viewer>
+            </template>
+          </example-snippet>
+        </div>
+      </div>
+    </div>
+
+    <div class="sample">
+      <div id="demo-container-5" class="demo"></div>
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">Cycling between posters</h2>
+            <h4></h4>
+          </div>
+          <example-snippet stamp-to="demo-container-5" highlight-as="html">
+            <template>
+  <model-viewer id="toggle-poster" reveal="interaction" auto-rotate poster="../../assets/poster-astronaut2.png" src="../../shared-assets/models/Astronaut.glb" alt="A 3D model of an astronaut"></model-viewer>
+  <script>
+    const posters = ['poster-astronaut2.png', 'poster-astronaut3.png', 'poster-astronaut4.png'];
+    const togglePoster = document.querySelector('#toggle-poster');
+    let i = 0;
+    setInterval(() =>
+      togglePoster.setAttribute('poster', `../../assets/${posters[++i % 3]}`), 2000);
+  </script>
+
+            </template>
+          </example-snippet>
+        </div>
+      </div>
+    </div>
+
+    <div class="footer">
+      <ul>
+        <li class="attribution">
+          <a href="https://poly.google.com/view/dLHpzNdygsg">Astronaut</a> by <a href="https://poly.google.com/user/4aEd8rQgKu2">Poly</a>,
+          licensed under <a href="https://creativecommons.org/licenses/by/2.0/">CC-BY</a>.
+        </li>
+
+        <li class="attribution">
+          <a href="https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/DamagedHelmet">Damaged Helmet</a> by <a href="https://sketchfab.com/theblueturtle_">theblueturtle_</a>,
+          licensed under <a href="https://creativecommons.org/licenses/by-nc/3.0/us/">Creative Commons Attribution-NonCommercial</a>.
+        </li>
+
+        <li class="attribution">
+          <a href="https://poly.google.com/view/6uTsH2jqgVn">Shish kebab</a> by <a href="https://poly.google.com/user/4aEd8rQgKu2">Poly</a>,
+          licensed under <a href="https://creativecommons.org/licenses/by/2.0/">CC-BY</a>.
+        </li>
+      </ul>
+
+      <div style="margin-top:24px;" class="copyright">©Copyright 2018-2020 Google Inc. Licensed under the Apache License 2.0.</div>
+      <div><a href="https://github.com/google/model-viewer">Github</a> ∙ <a href="https://model-viewer.glitch.me/">Glitch</a> ∙ <a href="https://github.com/google/model-viewer/issues">Bug report</a></div>
+  </div>
+      
+</div>
+
+</div>
+
+<script type="module" src="../../examples/built/docs-and-examples.js">
+</script>
+<script type="module">
+  (() => { init('examples/loading'); })();
```

Comment:
Presumably this addition is why these new file copies are here? And once we do the final docs merge, we'll replace the existing examples with these versions?

Reply:
yep! keeping two copies right now so nothing breaks -- necessary for testing